src/utils/PermissionUtils.js

```diff
@@ -15,9 +15,9 @@ module.exports = class PermissionUtils {
   }
 
   static isManager (client, user) {
-    const botGuild = client.guilds.get(process.env.BOT_GUILD)
-    const managerRole = botGuild && botGuild.roles.get(process.env.MANAGER_ROLE)
-    const isManager = (managerRole && managerRole.members.has(user.id)) || this.isDeveloper(client, user)
+    const botGuild = client.guilds.cache.get(process.env.BOT_GUILD)
+    const managerRole = botGuild && botGuild.roles.cache.get(process.env.MANAGER_ROLE)
+    const isManager = (managerRole && managerRole.members.cache.has(user.id)) || this.isDeveloper(client, user)
     return isManager
   }
```

Comment:
won't work cause the guild might not be on the same shard

Reply:
yea, thats expected

Reply:
O boy, I know nothing about sharding.

Reply:
that will also happen with all profile, leaderboard, userinfo and guildinfo